### ISSUE TITLE
hackjcvd processing

### ISSUE THREAD

**Issue by scenaristeur** [Phonations/HackJCVD#7]
Hein hein !!! 
 t'avais déjà réfléchi au problème : https://github.com/fhacktory/hackjcvd/tree/master/panel_step_one

**Comment by MartinDelille** [comment]
Oui, j'avais créer une ébauche de projet avec une liste de fonctionnalité. On a pas pu la reprendre car le fHackory interdit la réutilisation de code (ce qui n'est pas une bonne idée à mon sens).
